### PR TITLE
Gautham/iOS resource (Requested changes done)

### PR DESCRIPTION
**Pull request by GauthamG2**
## What does this PR do?
Add resource(s) | Remove resource(s) | Add info | Improve repo

## Checklist:
- [ ] Read our [contributing guidelines](https://github.com/EbookFoundation/free-programming-books/blob/main/docs/CONTRIBUTING.md).
- [ ] [Search](https://ebookfoundation.github.io/free-programming-books-search/) for duplicates.
- [ ] Include author(s) and platform where appropriate.
- [ ] Put lists in alphabetical order, correct spacing.
- [ ] Add needed indications (PDF, access notes, under construction).
- [ ] Used an informative name for this pull request.

## Follow-up

- Check the status of GitHub Actions and resolve any reported warnings!

### CODE REVIEW

File: books/free-programming-books-en.md [hunk 1]
```diff
@@ -1,10 +1,15 @@
...
+* [Swift](#swift)
 
 ### All
 
 * [English, By Programming Language](free-programming-books-langs.md)
 * [English, By Subject](free-programming-books-subjects.md)
   (The list of books in English is here for historical reasons.)
+
+
+### Swift
```
Comment (shrn01) [inline books/free-programming-books-en.md:13]: Please move this change to the english books file free-programming-books-langs.md

File: courses/free-courses-en.md [hunk 2]
```diff
@@ -1281,7 +1281,10 @@
 * [Server-Side Swift](https://www.udacity.com/course/server-side-swift--ud1031) - Jarrod Parkes, Nic Jackson (Udacity)
 * [Swift 5 for Beginners](https://www.youtube.com/playlist?list=PL5PR3UyfTWvfacnfUsvNcxIiKIgidNRoW) - iOS Academy
 * [Swiftris - Build an iOS Tetris app from scratch](https://www.bloc.io/swiftris-build-your-first-ios-game-with-swift)
-
+* [100 days of SwiftUI](https://www.hackingwithswift.com/100/swiftui) - Hacking With Swift UI
+* [What Is Swift UI? Easy Steps Building Your first SwiftUI app](https://www.udemy.com/course/what-is-swift-ui-part-1/) - Matthew Harding (Udemy)
+* [Build Great IOS Apps (Swift)](https://www.udemy.com/course/build-great-ios-apps-with-swift/?ranMID=39197&ranEAID=JVFxdTr9V80&ranSiteID=JVFxdTr9V80-ApSDs.hyUFklgG3FVVuVLA&LSNPUBID=JVFxdTr9V80&utm_source=aff-campaign&utm_medium=udemyads) - Hamad Fouad (Udemy)
+* [iOS Development Course - Use Swift 5 and UIKit to Build a Netflix Clone](https://www.youtube.com/watch?v=KCgYDCKqato) - freeCodeCamp.org (Youtube)
```
Comment (shrn01) [inline courses/free-courses-en.md:1287]:
```suggestion
* [iOS Development Course - Use Swift 5 and UIKit to Build a Netflix Clone](https://www.youtube.com/watch?v=KCgYDCKqato) - freeCodeCamp.org
```
Youtube attribution is not required

File: courses/free-courses-en.md [hunk 3]
```diff
@@ -1281,7 +1281,10 @@
 * [Server-Side Swift](https://www.udacity.com/course/server-side-swift--ud1031) - Jarrod Parkes, Nic Jackson (Udacity)
 * [Swift 5 for Beginners](https://www.youtube.com/playlist?list=PL5PR3UyfTWvfacnfUsvNcxIiKIgidNRoW) - iOS Academy
 * [Swiftris - Build an iOS Tetris app from scratch](https://www.bloc.io/swiftris-build-your-first-ios-game-with-swift)
-
+* [100 days of SwiftUI](https://www.hackingwithswift.com/100/swiftui) - Hacking With Swift UI
+* [What Is Swift UI? Easy Steps Building Your first SwiftUI app](https://www.udemy.com/course/what-is-swift-ui-part-1/) - Matthew Harding (Udemy)
+* [Build Great IOS Apps (Swift)](https://www.udemy.com/course/build-great-ios-apps-with-swift/?ranMID=39197&ranEAID=JVFxdTr9V80&ranSiteID=JVFxdTr9V80-ApSDs.hyUFklgG3FVVuVLA&LSNPUBID=JVFxdTr9V80&utm_source=aff-campaign&utm_medium=udemyads) - Hamad Fouad (Udemy)
```
Comment (shrn01) [inline courses/free-courses-en.md:1286]:
```suggestion
* [Build Great IOS Apps (Swift)](https://www.udemy.com/course/build-great-ios-apps-with-swift) - Hamad Fouad (Udemy)
```

File: courses/free-courses-en.md [hunk 4]
```diff
@@ -1281,7 +1281,10 @@
 * [Server-Side Swift](https://www.udacity.com/course/server-side-swift--ud1031) - Jarrod Parkes, Nic Jackson (Udacity)
 * [Swift 5 for Beginners](https://www.youtube.com/playlist?list=PL5PR3UyfTWvfacnfUsvNcxIiKIgidNRoW) - iOS Academy
 * [Swiftris - Build an iOS Tetris app from scratch](https://www.bloc.io/swiftris-build-your-first-ios-game-with-swift)
-
+* [100 days of SwiftUI](https://www.hackingwithswift.com/100/swiftui) - Hacking With Swift UI
```
Comment (shrn01) [inline courses/free-courses-en.md:1284]: This is already added in 1277

**Review by shrn01 — COMMENTED** [review]
Please make the requested changes

**Comment by GauthamG2** [comment]
Final change also done. Removed the course from the list.

File: books/free-programming-books-en.md [hunk 5]
```diff
@@ -7,4 +7,7 @@
 
 * [English, By Programming Language](free-programming-books-langs.md)
 * [English, By Subject](free-programming-books-subjects.md)
+* [Swift algorithms and data structures](https://englishonlineclub.com/pdf/Data%20Structures%20and%20Algorithms%20in%20Swift%20-%20Implementing%20practical%20data%20structures%20with%20Swift%204%20[EnglishOnlineClub.com].pdf) (PDF)
```
Comment (shrn01) [inline books/free-programming-books-en.md:10]: Please remove the changes from this file

**Review by shrn01 — CHANGES_REQUESTED** [review]

**Comment by GauthamG2** [comment]
Changes removed from the file.

File: books/free-programming-books-en.md [hunk 6]
```diff
@@ -8,3 +8,5 @@
 * [English, By Programming Language](free-programming-books-langs.md)
 * [English, By Subject](free-programming-books-subjects.md)
   (The list of books in English is here for historical reasons.)
+
```
Comment (shrn01) [inline books/free-programming-books-en.md:11]: These new lines too

**Comment by GauthamG2** [comment]
I didn't add these, but I removed them now.

**Comment by GauthamG2** [comment]
Done, Please check now.

**Comment by GauthamG2** [comment]
The PR/MR is still in NOT ACCEPTED state in my Hacktobefest profile. Could
you please check it.

**Comment by GauthamG2** [comment]
The PR/MR is still in NOT ACCEPTED state in my Hacktobefest profile. Could
you please check it

File: books/free-programming-books-en.md (hunk 6 above)
Comment (eshellman) [inline books/free-programming-books-en.md:11]:
```suggestion
```

File: books/free-programming-books-en.md [hunk 7]
```diff
@@ -8,3 +8,5 @@
...
+
```
Comment (eshellman) [inline books/free-programming-books-en.md:12]:
```suggestion
```

File: books/free-programming-books-en.md [hunk 8]
```diff
@@ -8,3 +8,4 @@
 * [English, By Programming Language](free-programming-books-langs.md)
 * [English, By Subject](free-programming-books-subjects.md)
   (The list of books in English is here for historical reasons.)
+
```
Comment (eshellman) [inline books/free-programming-books-en.md:11]:
```suggestion
```

**Comment by GauthamG2** [comment]
Thanks for merging the commit. But in the Hacktoberfest profile page, this
PR/MR is still in NOT ACCEPTED state.

**Comment by shrn01** [comment]
Hey @GauthamG2 hacktoberfest is done. The time limits are October only. This pr was merged in Nov 3. So that's why you're not seeing it

**Pull request merged**